texcount.py

```diff
@@ -55,8 +57,13 @@ def run(self, edit, **args):
             )
 
             result = p.communicate()[0].decode('utf-8').strip()
+            # result = p.communicate()[1].decode('utf-8').strip()
             if p.returncode == 0:
-                self.view.window().show_quick_panel(result.splitlines()[1:-4], None)
+                res_split = result.splitlines()
+                try:
+                    self.view.window().show_quick_panel(res_split[1:4] + res_split[9:], None)
+                except TypeError:
+                    self.view.window().show_quick_panel(res_split[1:4], None)
```

Comment:
I'm not sure I understand what this line is supposed to do. Admittedly I wasn't checking for a TypeError before, but if its raised on 64, surely this will just re-raise a different instance of the same exception?